br/pkg/restore/client.go

```diff
@@ -1251,6 +1251,43 @@ func (rc *Client) ReadStreamDataFiles(ctx context.Context, metas []*backuppb.Met
 	return streamBackupDataFiles, nil
 }
 
+// FixIndex tries to fix a single index.
+func (rc *Client) FixIndex(ctx context.Context, schema, table, index string) error {
+	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
+		span1 := span.Tracer().StartSpan(fmt.Sprintf("Client.LoadRestoreStores index: %s",
+			strings.Join([]string{schema, table, index}, ".")), opentracing.ChildOf(span.Context()))
```

Comment:
ok why not just make it `fmt.Sprintf("Client.LoadRestoreStores index: %s.%s.%s", schema, table, index)`